prac_05/hex_colours.py

```diff
@@ -0,0 +1,36 @@
+"""Hex colours program: Enter a colour name and receive its corresponding hex colour."""
+COLOUR_HEX_DICT = {"firebrick": "#b22222", "DodgerBlue1": "#1e90ff", "DeepPink1": "#ff1493", "DarkSlateBlue": "#483d8b",
+                   "DarkOrange": "	#ff8c00", "DarkOliveGreen1": "#caff70", "DarkGoldenrod": "#b8860b",
+                   "coral": "#ff7f50",
+                   "black": "#000000", "AntiqueWhite": "#faebd7"}
```

Comment:
Space before code for dark orange can be removed. Black and AntiqueWhite can be moved to previous line.

Reply:
PEP8 standards suggest not to exceed character count; hence the odd spacing.